<various>: Split "Firewalls" section into distinct sections.

flexibeast opened the pull request:
At request of @the-maldridge, to facilitate expanding content.

@ericonr: Does this look okay for me to merge?

ericonr reviewed (commented):
Would it be too ugly to put the Firewall page inside `network/index.md`?

flexibeast commented:
@ericonr: Hmm, no, i don't think it would be .... Whereabouts would you suggest?

flexibeast commented:
@ericonr: i've added the contents of "Firewalls" to the end of `network/index.md`, and removed `firewalls.md`. Let me know if you think the content should go elsewhere on the page.

Duncaen commented:
Is this really worth it, is there that much content planned?

flexibeast commented:
@Duncaen: You'd have to ask @the-maldridge. :-)
